src/api/app/models/project.rb

```diff
@@ -79,6 +79,7 @@ def autocomplete(search)
 
   has_one :staging, class_name: 'StagingWorkflow', inverse_of: :project
   belongs_to :staging_workflow, inverse_of: :staging_projects
+  has_and_belongs_to_many :bs_requests
```

Comment:
This really deserves a more specific term - projects can have tons of relations to requests. Source action, target action, review, ... - but this is purely about selected into a staging project?